zuul.d/jobs.yaml

```diff
@@ -79,6 +79,21 @@
     vars:
       tox_envlist: py35
       python_version: 3.5
+- job:
+    name: tox-py39
+    parent: tox
+    description: |
+      Run unit tests for a Python project under cPython version 3.9.
+
+      Uses tox with the ``py39`` environment.
+
+    nodeset:
+      nodes:
+        - name: impish-medium
+          label: impish-medium
+    vars:
+      tox_envlist: py39
+      python_version: 3.9
```

Comment:
it may need to be "3.9" (quoted) as opposed to 3.9 as that might come through as a float in yaml-speak.

Reply:
`python_version: 3.5` in the previous definition is not quoted, so I assumed the parser was going to do the right thing :thinking: